index.js

```diff
@@ -66,7 +80,7 @@ genderView(usersArray);
 const data = genderView(usersArray);
 
 const genderCount = data => {
-  // Your code goes here ...
+  console.log(`Female: ${data.femaleUsers.length}\nMale: ${data.maleUsers.length}`)
```

Comment:
a console log does not "return" a value. this function would run something in a console. for this exercise its cool, but ideally try to return instead :)